package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@react-native-community/checkbox",
+  "name": "react-native-checkbox",
```

Comment:
please remain this `"name": "@react-native-community/checkbox"` just like another repo of react-native-community